fw_pipeline_io/fw_data_tools/file_downloader.py

```diff
@@ -118,3 +119,19 @@ def download_and_unzip_files(
                 self.unzip_files(file_path, downloaded_folder)
 
         return downloaded_folder
+
+
+class PrepareSync:
+    @staticmethod
+    def add_uuid_tag_to_container(container, uuid_tag: str = None):
+        if uuid_tag is None:
+            uuid_tag = str(uuid4().hex)  # meet Flywheel's 32 character tag limit
```

Comment:
Clarify in comment that adding .hex is what makes uuid4()'s output meets the 32 character limit.

Reply:
added clarification